test/clj/rems/db/testing.clj

```diff
@@ -6,14 +6,16 @@
             [rems.api.applications-v2]
             [rems.config :refer [env]]
             [rems.db.core :as db]
+            [rems.db.dynamic-roles]
             [rems.db.test-data :as test-data]))
 
 (defn test-db-fixture [f]
   (mount/stop) ;; during interactive development, app might be running when tests start. we need to tear it down
   (mount/start-with-args {:test true}
                          #'rems.config/env
                          #'rems.db.core/*db*
-                         #'rems.api.applications-v2/all-applications-cache)
+                         #'rems.api.applications-v2/all-applications-cache
+                         #'rems.db.dynamic-roles/dynamic-roles-cache)
```

Comment:
so are the caches enabled in tests to test the cache code? it would seem simpler to run most test without caching

Reply:
Yes, it helps test the caching. How about we have a `caches-fixture` which starts all the caches and which would be used in the API tests? Or would it be better to test caching separately?

Reply:
not sure what's best, but perhaps let's start with a `caches-fixture` that's used in the api tests